galaxyproject/galaxy
# Only Render View Report and Generate PDF Buttons for Successful Workflows.

Fix for #16922 

Only display View Report or Generate Report buttons if the workflow invocation and steps were succcessful
Simply added a v-if to those buttons and a method to checks invocation status and step status

![Screenshot from 2023-12-06 18-36-24](https://github.com/galaxyproject/galaxy/assets/26912553/cd536b73-3429-43aa-9424-265bc3d3ce25) r

## How to test the changes?
(Select all options that apply)
  1. Navigate to User > Workflow Invocations
  2. Note that you can see & click "View Report" and "Generate PDF" for successful Workflows, but not for unsuccessful ones.
 

## License
